src/launcher/src/main/resources/application.properties

```diff
@@ -28,21 +28,24 @@ management.endpoint.prometheus.enabled=true
 
 ###############################################################################
 # Application setup
-application.setup.admin.username=admin@everest.engineering
-application.setup.admin.password=ac0n3x72
-application.oauth.client.id=web-app-ui
-application.oauth.client.secret=replace-me
-application.jwt.signing-secret=replace-this-with-something-really-secret
-application.jwt.access-token.validity-seconds=36000
-application.jwt.refresh-token.validity-seconds=360000
 application.axon.retry.interval-milli-seconds=1000
 application.axon.retry.max-count=1
 application.axon.retry.pool-size=1
 application.axon.event-processor.default-group=true
 application.axon.event-processor.segments=1
-application.security.endpoint.matchers.anonymous=/api/organizations/register/**,/api/organizations/**/register/**,/api/version,/actuator/health/**,/api/doc/**,/swagger-ui/**,/swagger-resources/**
-application.security.endpoint.matchers.authenticated=/api/**,/actuator/prometheus/**,/**
-application.security.endpoint.matchers.admin=/admin/**,/actuator/**
+
+keycloak.enabled=true
+keycloak.realm=default
+keycloak.resource=default-client
+keycloak.auth-server-url=http://localhost:8180/auth
+keycloak.credentials.secret=replace-me
+keycloak.use-resource-role-mappings=true
+keycloak.confidential-port=0
+
+kc.server.admin-user=admin@everest.engineering
```

Comment:
Two different roots are being used here: 'keycloak' and 'kc'. Should be consistent and make them all 'keycloak'.

Reply:
We can't make all of them as `keycloak`. `keycloak` expects only predefined properties otherwise throws an error at runtime.

Reply:
Urgh. Yuck.